src/app/store/slices/user/index.ts

```diff
@@ -128,6 +129,7 @@ export const logoutThunk = createAsyncThunk<void, void, { state: RootState }>(
     dispatch(storageActions.resetState());
     dispatch(uiActions.resetState());
     dispatch(referralsActions.resetState());
+    dispatch(workspacesActions.resetState());
```

Comment:
Seeing this comes to my mind, does it also clean the workspace data in the local storage?